### DISCUSSION TITLE
Consulta Compejidad espacial

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#227] [Preguntas y respuestas] [answered]
Hola, buenas tardes, tengo una duda  respecto a una funcion con retorno.

Teniendo por ejemplo la funcion: 
```C
int serie_fibonacci(int numero)
{
	int sumador1 = 0;
	int sumador2 = 1;
	int digito_fibonacci;
	int suma_final = 0;

	for(int i = 0; i < numero; i++)
	{
		digito_fibonacci = sumador1 + sumador2;
		suma_final = suma_final + digito_fibonacci;
		sumador1 = sumador2;
		sumador2 = digito_fibonacci;
	}
	return digito_fibonacci;
}
```

la linea `return digito_fibonacci` sumaria 1 tambien ? a parte de el resto de los `int`?

**Comment by sanvaltos** [comment, marked as answer]
En el ejemplo que esta en el TP10 se puede ver que se toma en cuenta el retorno a la hora de realizar el calculo de complejidad espacial. Y en mi caso estoy sumando 1 al hacer las cuentas de complejidad temporal por cada return.
